### DISCUSSION TITLE
uWS/uWS.h

### DISCUSSION THREAD

**Discussion by skitaeff** [uNetworking/uWebSockets#1082] [Q&A] [answered]
The software that I'm trying to use is looking for <uWS/uWS.h> from this library, which I can't seem to find anywhere. Is it something's been deprecated? Does it still exist in a specific release?

**Comment by ghost** [comment, marked as answer]
That's v0.14 and yes it's deprecated

**Reply by Bestbbb** [reply]
It means I use v0.14 now and it will not work out.Isn't it?I try to review a project 3years ago using v0.14 and it is not useful.
